src/main/java/com/neighborhoodAssurance/NeighborController.java

```diff
@@ -1,15 +1,10 @@
 package com.neighborhoodAssurance;
 
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.*;

```

Comment:
Best to keep this as it was.  The wildcard operator is generally shunned in Java, because 1) explicit imports are good for documentation... you know why the import is happening, 2) disambiguation... if you import java.sql.* and java.util.*, and you reference a type Date, is it java.sql.Date or java.util.Date?  Both packages have classes named Date, and 3) the IDE/compiler will optimize these for you.